Mlem/App/Views/Shared/Images/Core/MediaView.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  MediaView.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-09-26.
+//
+
+import SDWebImageSwiftUI
+import SwiftUI
+import UIKit
+
+struct MediaView: View {
+    let media: MediaType
+    
+    @Binding var playing: Bool
```

Comment:
Doesn't need to be a `Binding` I don't think
```suggestion
    let playing: Bool
```

Reply:
Good catch, holdover from an earlier iteration